.jest/setEnvVars.ts

```
@@ -1 +1,13 @@
-process.env.NODE_ENV = 'test';
+import 'reflect-metadata'
+
+process.env.NODE_ENV = 'test'
+
+// Set CI-friendly timeouts and limits
+if (process.env.CI) {
```

ESLint

[error] Unexpected use of the global variable 'process'. Use 'require("process")' instead.

(node/prefer-global/process)